ios-base/Gemfile

```diff
@@ -0,0 +1 @@
+gem 'cocoapods'
```

Comment:
Can you please add *gem source* and ruby version? You can copy it from `.ci/Gemfile`.

Reply:
I did it👍